src/coldfront_plugin_api/utils.py

```diff
@@ -48,3 +48,6 @@ def get_or_fetch_user(username):
         user = create_user(**found)
 
     return user
+
+def SCIM_is_authenticated_predicate(user: User):
```

Comment:
A better name for this function is `is_user_superuser(user: User) -> bool` as that describes what this function is doing in this specific case, and therefore when you change the setting you get ```'GET_IS_AUTHENTICATED_PREDICATE': 'coldfront_plugin_api.utils.is_user_superuser'``` which describes the configuration setting without having to dig into the code.